packages/EtendoHookBinder/src/api/defaultConfig.ts

```diff
@@ -0,0 +1,27 @@
+import { API_SET_DEFAULT_CONFIGURATION_URL } from './constants';
+import { DefaultConfiguration } from './types';
+
+export const setDefaultConfiguration = async (token: string, config: DefaultConfiguration): Promise<void> => {
+  const response = await fetch(API_SET_DEFAULT_CONFIGURATION_URL, {
```

Comment:
Please use the existing api client in `packages/EtendoHookBinder/src/api/client.ts`
We need to avoid executing requests outside of the API client. 

You could do:
```ts

const client = new Client(API_SET_DEFAULT_CONFIGURATION_URL).setAuthHeader(token, 'Bearer');
const params = new URLSearchParams({
  language: config.language || '192',
  role: config.defaultRole,
  client: config.client || 'System',
  organization: config.organization,
  warehouse: config.defaultWarehouse,
  default: true,
});
const response = await client.post('/', params);
```